src/app/dashboard/org/actions.ts

```
@@ -15,6 +15,7 @@ import {
 } from "@/lib/permissions";
 import { revalidatePath } from "next/cache";
 import { Prisma } from "@prisma/client";
+import { checkPermission } from "@/lib/permission";
```

ESLint

[error] 'checkPermission' is defined but never used.

(@typescript-eslint/no-unused-vars)